plugins/bgp/GlobalConfigurator/plugin_globalconf.go

```diff
@@ -0,0 +1,27 @@
+package GlobalConfigurator
+
+import (
+	"github.com/contiv/bgp-vpp/plugins/bgp/model"
+)
+
+// CreateGlobalConf creates a new plugin in etcd
+func (b *GlobalConfHandler) CreateGlobalConf(val *model.GlobalConf) error {
+	err := b.broker.Put(val.GetName(), val)
+	if err != nil {
+		b.log.Errorf("Could not create configuration")
+		return err
+	}
+
+	return nil
+}
+
+// DeleteGlobalConf deletes a plugin in etcd
+func (b *GlobalConfHandler) DeleteGlobalConf(key string) error {
+	existed, err := b.broker.Delete(key)
+	if err != nil {
+		b.log.Errorf("Could not delete configuration")
+	}
+	b.log.Infof("Configuration existed: %v", existed)
+
+	return nil
+}
```

Comment:
ah, I see what you're doing here - you're trying to put the data back to etcd. But that's where we just got it from through the KVScheduler. Rather, you need to push the received data into the BGP Server, as i outlined on Slack